README.md

```diff
@@ -41,9 +41,51 @@ npm run dev
 ### 1단계 - 웹 성능 테스트
 1. 웹 성능예산은 어느정도가 적당하다고 생각하시나요
 
-2. 웹 성능예산을 바탕으로 현재 지하철 노선도 서비스는 어떤 부분을 개선하면 좋을까요
-
+#### 타사 비교 (First View, 데스크톱)
+사이트 3개 모두 TTI, TBT는 PageSpeed (데스크톱), 그 외 메트릭은  webpage test 수치 사용.
+단위: 초(s)
+||[네이버 지도](https://map.naver.com/v5/directions/-/-/-/transit?c=14137575.5199888,4524330.9973991,15,0,0,0,dh)|[카카오지도]()|[nextstep subway](https://dibtp1221.kro.kr/)|
+|----------------|-------------------------------|-----------------------------|-----------------------------|
+|FCP|0.389|0.741|4.853|
+|TTI|3.2|2.5|2.8|
+|Speed Index|1.931|3.878|4.803|
+|LCP|3.975|2.141|4.885|
+|TBT|0.021|0.059|0.005|
+
+#### 타사 비교 (Repeat View, 데스크톱)
+사이트 3개 모두 메트릭 webpage test 수치 사용.
+단위: 초(s)
+||[네이버 지도](https://map.naver.com/v5/directions/-/-/-/transit?c=14137575.5199888,4524330.9973991,15,0,0,0,dh)|[카카오지도]()|[nextstep subway](https://dibtp1221.kro.kr/)|
+|----------------|-------------------------------|-----------------------------|-----------------------------|
+|FCP|0.242|0.463|0.639|
+|Speed Index|0.761|1.481|0.700|
+|LCP|1.406|0.718|0.695|
+|TBT|>=0.408|>=0.873|>=0.000|
+
+
+#### First View 웹 성능 예산 [nextstep subway](https://dibtp1221.kro.kr/)
+||목표|
+|----------------|-------------------------------|
+|FCP|0.7|
+|TTI|2|
+|Speed Index|2|
+|LCP|1|
+|TBT|유지|
+이렇게 정한 이유 개선 필요한 부분과 함께 설명
 
+2. 웹 성능예산을 바탕으로 현재 지하철 노선도 서비스는 어떤 부분을 개선하면 좋을까요
+- gzip 압축
+    - 웹 페이지 처음 로드할때 다운로드 받는 컨텐츠들을 보면 JS 파일이 2개 총 request 수 중에 11.1% 를 차지하지만 총Bytes에서는 2,352,130 (92.1%) 를 차지한다. 이것은 JS 파일이 gzip 압축이 안 되어있어서 그렇다.
+    - 네이버, 카카오 등 타 서비스를 보면 gzip compression score 100/100을 달성함에 비해, 우리 서비스는 22/100이고, vendor.js 와 main.js 가 압축이 안되어있다.
+    - vendor.js (2,125KB -> 407KB) , main.js (172.0 -> 28.1KB) 로 각각 압축이 가능하며 그렇게 되면 일단 FCP와 LCP를 1/4 정도로는 줄일 수 있다. 그럼 1~2 값이 되고
+
+- CDN
+    - WAS 서버에서 직접 .js 와 .png 파일을 내려주는 대신에 CDN 서버를 이용하면 클라이언트가 더 빠르게 자원을 받을 수 있을거로 생각한다.
+    - 그렇게 되면 First View FCP와 LCP를 각각 0.7, 1 정도 수치까지 내릴 수 있지 않을까 생각했다.
+
+- 컨텐츠에 브라우저 캐시 사용을 위한 헤더 추가
```

Comment:
[질문] 우리 서비스 수치를 보면 first view 와 repeat view 의 차이가 엄청 크고,
repeat view 에서 컨텐츠를 새로 받지 않고 first 에서 받을 걸 써서 그렇다고 생각했습니다!

현재 subway 자체 컨텐츠들은 브라우저 캐시 설정이 거의 되어있지 않은데요.
그래도 repeat view 에서 수치에 급격히 좋아지는 이유는
같은 클라이언트가 그냥 연속적으로 로드해서 그런거고, 브라우저를 종료 후 다시 들어온다면 브라우저 캐시를 쓸 수 없을까요??

Reply:
음.. 이 부분은 잘 모르겠네요.

<img width="1440" alt="스크린샷 2022-04-30 오후 11 07 57" src="https://user-images.githubusercontent.com/29951288/166108932-5f5c2886-5653-4b9e-8f02-6b19554283f9.png">
테스트 결과를 보면, 자바스크립트, 이미지, 폰트 등을 재요청하지 않아서 그렇다고 볼 수 있는데요. 왜 재요청을 하지 않는지까지는 모르겠네요.


> Repeat View:
The Repeat View row is a test that was done immediately after the First View test without clearing out anything. The browser window is closed after the First View test and then a new browser is launched to do the Repeat View test. The Repeat View test represents what someone will see if they are coming back to the page some time after visiting it the first time (and is a best-case scenario since the test is re-visiting the page immediately).

Repeat View 기능은 브라우저를 끄고 다시 실행하는 것이니 브라우저 캐시가 걸릴 일이 없고, 
이와 별개로, 현재 서버는 말씀하셨듯 캐싱 설정이 되어 있지 않고, 요청할 때도 no-cache 요청을 하기에 매번 재로딩 됩니다.

브라우저에서 직접 퍼포먼스 탭으로 보았을 떄도 브라우저 캐싱이 안되는 것을 확인할 수 있어요

<img width="1383" alt="스크린샷 2022-04-30 오후 11 23 06" src="https://user-images.githubusercontent.com/29951288/166109517-43a8ae41-ef4e-45c1-b882-43acf6f5e65c.png">

Reply:
우와..!! 감사합니다! 브라우저 퍼포먼스 탭 있는 거 알았어도
잘 활용을 못했었는데
다음에는 이런 확인이 필요할때 활용해보도록 하겠습니다. 감사합니다!!